ansible/roles/common/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+common:
+  upgrade_base: False
```

Comment:
Why does this default to `False` when we explicitly always set it to `True`? Seems like `True` should be the default and we can nix including it, or include it for explicitness. I think not wanting to upgrade the base is the exception not the norm, but I am not in the business of building images (usually).

Reply:
This defaults to False, because if this playbook is run multiple times, it may have unexpected consequences.  We discussed this internally and are using this in order to bridge the gap between upstream image packages and any delta that may exist when we are building our own images.